src/main/java/com/qaprosoft/carina/demo/mobile/gui/pages/android/LoginPage.java

```diff
@@ -86,31 +92,37 @@ public CarinaDescriptionPageBase login() {
 
     @Override
     public boolean isPageOpened() {
+        wait.until(ExpectedConditions.presenceOfElementLocated(By.id("login_button")));
```

Comment:
isElementPresent() has waiter, it doesn’t need waiter here

Reply:
done